src/ncurses/window.cr

```diff
@@ -23,6 +24,26 @@ module NCurses
       end
     end
 
+    def self.subwin(height, width, y, x, parent, &blk)
+      win = new(LibNCurses.subwin(parent, height, width, y, x))
+      wind.closed = false
```

Comment:
`win.closed = false` ?
`@closed` is `false` by default, so I guess this line is not necessary.